common/utils.mjs

```diff
@@ -22,46 +22,8 @@ export function kebabCaseToPascalCase (string) {
     .join('');
 }
 
-/**
- * checks whether the dt-scrollbar is being used on the root element.
- * @param rootElement {HTMLElement}
- * @returns {boolean}
- */
-function isDtScrollbarInUse (rootElement = document.documentElement) {
-  if (rootElement.hasAttribute('data-overlayscrollbars')) {
-    return true;
-  }
-  return false;
-}
-
-/**
- * This will disable scrolling on the root element regardless of whether you are using dt-scrollbar or not.
- * @param rootElement {HTMLElement}
- */
-export function disableRootScrolling (rootElement = document.documentElement) {
-  if (isDtScrollbarInUse(rootElement)) {
-    rootElement.classList.add('d-scrollbar-disabled');
-  } else {
-    rootElement.classList.add('d-of-hidden');
-  }
-}
-
-/**
- * This will enable scrolling on the root element regardless of whether you are using dt-scrollbar or not.
- * @param rootElement {HTMLElement}
- */
-export function enableRootScrolling (rootElement = document.documentElement) {
-  if (isDtScrollbarInUse(rootElement)) {
-    rootElement.classList.remove('d-scrollbar-disabled');
-  } else {
-    rootElement.classList.remove('d-of-hidden');
-  }
-}
```

Comment:
Should we have some rule for this sort of thing, when should it go in common/utils vs dialtone-vue?

Should we not have common/utils at all? Or should we have a separate package for common utils?

Reply:
Yes, I thought moving everything to a common package was the way to go but as we're using relative paths, that messes up the build process.

If we move those utility functions to a `@dialpad/dialtone-utilities" package, would be possible to reference them and move common utilities across our monorepo easily, could also be an external dependency, tree-shakeable and everything.

Reply:
I think that's probably the way to go, but not necessary for this PR.